scene/main/viewport.cpp

```diff
@@ -1525,13 +1542,14 @@ void Viewport::_gui_show_tooltip() {
 		return;
 	}
 
-	Control *rp = which;
-
-	Control *base_tooltip = which->make_custom_tooltip(tooltip);
+	// Controls can implement `make_custom_tooltip` to provide their own tooltip.
+	// This should be a Control node which will be added as child to a TooltipPanel.
+	Control *base_tooltip = which->make_custom_tooltip(tooltip_text);
```

Comment:
Perhaps should be named `base_tooltip_control` just to follow suit.

Reply:
I'm not sure this would be really useful as it's only used locally and it's clear from it's type declaration. This would also open the door to renaming `make_custom_tooltip` and tons of tooltip-related methods which can either be about tooltip widgets or tooltip texts in the existing API.

Reply:
Yeah. you are right. Better not go down the rabbit hole.